app/components/moderation-list-row/component.js

```diff
@@ -40,6 +40,10 @@ export default Ember.Component.extend({
         return this.get('submission.node.contributors.content.meta.pagination.total') - 3;
     }),
 
+    lastAction: Ember.computed('submission.actions', function(){
```

Comment:
Also, this needs to handle accepted submissions with no actions, and display something like "accepted automatically on {{date}}" (thoughts on language, @laurenbarker?)